packages/schematics/src/collection/application/index.ts

```diff
@@ -306,7 +311,7 @@ export default function(schema: Schema): Rule {
 
       excludeUnnecessaryFiles(),
 
-      move(options.e2eProjectName, options.e2eProjectRoot),
+      move('e2e', options.e2eProjectRoot),
```

Comment:
Something changed in the angular application schematics and because `projectRoot` is set to a falsey value (`''`) in the `angular.json`, the `e2e` project always gets generated to `e2e` directory..

Reply:
Thank you so much. I have applied the tweaked the fix a bit and applied it.